### PR TITLE
Update NIST templates workflow

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
*In relation to https://github.com/metanorma/metanorma/issues/228*

### CODE REVIEW

**Comment by ronaldtse** [comment]
@CAMOBAP can you also help fix this? Thanks!

**Comment by CAMOBAP** [comment]
@manuel489 thanks for the help with GitHub actions update, but for now we are in the middle of 'big' migration to "consolidated" workflows which will be stored in a single place:
 - https://github.com/metanorma/metanorma-build-scripts/blob/main/.github/workflows/template-docker.yml
 - https://github.com/metanorma/metanorma-build-scripts/blob/main/.github/workflows/template-test.yml
 - https://github.com/metanorma/metanorma-build-scripts/blob/main/.github/workflows/template-pages.yml

Check out https://github.com/metanorma/mn-templates-nist/pull/17

As far as I see this PR also have other changes which are probably important

I understand that our CI may look complicated, so please don't hesitate to ask me if you will need any change in it

**Comment by ronaldtse** [comment]
@CAMOBAP since @manuel489 is our editor, he would need your help to fix the actions. 😉 

The main task here in the PR is to add the site building manifest. 

Could you help fix the actions and merge this? Thanks!

**Comment by CAMOBAP** [comment]
> since @manuel489 is our editor, he would need your help to fix the actions. 😉
> Could you help fix the actions and merge this? Thanks!

actions are fixed now, in main branch

> The main task here in the PR is to add the site building manifest.

Yep, I will work on metanorma-cli to support template repos we have an issue described here https://github.com/metanorma/mn-templates-ogc/pull/46

**Comment by manuelfuenmayor** [comment]
Thanks @CAMOBAP, I see that the build failure is solved in main branch. So, I agree on closing this PR 👍.

**Comment by ronaldtse** [comment]
The content fixes are not fixed yet, let me rebase.

File: sp-800/sections/99-references.adoc [hunk 1]
```diff
@@ -49,25 +49,25 @@ https://doi.org/10.6028/NIST.IR.8179
 * [[[csf,NIST CSF]]],	National Institute of Standards and Technology (2018) _Framework for Improving Critical Infrastructure Cybersecurity, Version 1.1_. (National Institute of Standards and Technology, Gaithersburg, MD). https://doi.org/10.6028/NIST.CSWP.04162018
 
 // A NIST cybersecurity white paper. Note that most of these are now being assigned DOIs.
-* [[[ref5,5]]],	Souppaya MP, Regenscheid AR, Cooper DA, Cooley D, Bean C, Jenkins M, Boyle M (2018) _Security Considerations for Code Signing_. (National Institute of Standards and Technology, Gaithersburg, MD). https://doi.org/10.6028/NIST.CSWP.01262018
+* [[[ref5,Boyle_2018]]],	Souppaya MP, Regenscheid AR, Cooper DA, Cooley D, Bean C, Jenkins M, Boyle M (2018) _Security Considerations for Code Signing_. (National Institute of Standards and Technology, Gaithersburg, MD). https://doi.org/10.6028/NIST.CSWP.01262018
```
Comment (ronaldtse) [inline sp-800/sections/99-references.adoc]: @manuel489 please watch out for 'tab' characters. Thanks!

Comment (ronaldtse) [reply sp-800/sections/99-references.adoc]: I've fixed it.

**Pull request merged**